libs/openchallenges/challenge/src/lib/challenge-overview/challenge-overview.component.ts

```diff
@@ -19,8 +19,8 @@ export class ChallengeOverviewComponent {
   organizationCards: OrganizationCard[] = MOCK_ORGANIZATION_CARDS;
   // mockTopics = ['breast', 'cancer'];
 
-  use_default(str: string) {
-    return str === '' ? 'Not available' : str;
+  use_default(str: string | null | undefined) {
```

Comment:
- Use camel case for function name in TypeScript (`useDefault` instead of `use_default`)
- Use a more descriptive name